ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -0,0 +1,49 @@
+const path = require('path');
+const os = require('os');
+const { asyncify } = require('asyncbox');
+const { logger, fs, mkdirp } = require('appium-support');
+const { exec } = require('teen_process');
+const xcode = require('appium-xcode');
+
+const log = new logger.getLogger('WDABuild');
+
+async function buildWebDriverAgent (xcodeVersion) {
+  // Get Xcode version
+  xcodeVersion = xcodeVersion || await xcode.getVersion();
+  log.info(`Building bundle for Xcode version '${xcodeVersion}'`);
+
+  // Clean and build
+  await exec('npx', ['gulp', 'clean:carthage']);
+  log.info('Running ./Scripts/build.sh');
+  let env = {TARGET: 'runner', SDK: 'sim'};
+  await exec('/bin/bash', ['./Scripts/build.sh'], {env});
```

Comment:
so what is the reason to not provide cwd explicitly?

Reply:
The directory that `./Scripts/build.sh` needs to be run from is the same as the current working directory. Shouldn't we only be setting `cwd` if it's different from the actual current working directory?

Reply:
nope, if we 100% sure that the current folder is always the root folder of WDA project

Reply:
Okay. I changed it to use the root directory for cwd and path resolves.